misc/compare_tasks.py

```diff
@@ -0,0 +1,116 @@
+#!/usr/bin/env python
+""" Script to compare tasks from pushes on different Treeherder instances"""
```

Comment:
This script has just been moved. It is not a new script. You can just rubber-stamp this.